Dockerfile

```diff
@@ -64,13 +65,17 @@ RUN set -x \
     && mkdir -p /taiga /usr/src/taiga-front-dist/dist/js/ \
     && pip install --no-cache-dir -r /usr/src/taiga-back/requirements.txt \
     && pip install --no-cache-dir j2cli \
+    && pip install --no-cache-dir taiga-contrib-ldap-auth-ext \
     && echo "LANG=en_US.UTF-8" > /etc/default/locale \
     && echo "LC_TYPE=en_US.UTF-8" > /etc/default/locale \
     && echo "LC_MESSAGES=POSIX" >> /etc/default/locale \
     && echo "LANGUAGE=en" >> /etc/default/locale \
     && locale-gen en_US.UTF-8 && dpkg-reconfigure locales \
     && locale -a
 
+# Configure SSL ( Required for the LDAP plugin )
+RUN echo "CipherString=DEFAULT@SECLEVEL=1" >> /etc/ssl/openssl.cnf
```

Comment:
I think it's probably a final solution. It just relaxes the security requirements for the ciphers that it will allow connections over. If we wanted to we could leave it out and it would force people to upgrade the cipher suite on their LDAP servers, but I don't think it is our place to do so. If they want to make their LDAP server more secure that is up to them.

If we left it out, it wouldn't technically be a "problem" so to speak with the container, it would just refuse to connect to any LDAP servers that aren't at certain minimum security level. I think, though, that it makes sense to do it like this.

Reply:
Ok, sure.